static/js/components/email/EmailCompositionDialog.js

```diff
@@ -254,10 +261,9 @@ export default class EmailCompositionDialog extends React.Component {
             onEditorStateChange={this.onEditorStateChange}
             toolbar={draftWysiwygToolbar}
           />
-          <div className="toolbar-below">
-            <div className="insert">Insert:</div>
-            {this.makeCustomToolbarButtons(RECIPIENT_VARIABLE_NAMES)}
-          </div>
+          {dialogType !== COURSE_EMAIL_TYPE
```

Comment:
We should only show recipient variables for email which is sent in bulk. Can you add an option to the `EmailConfig` to specify whether each type supports bulk email or not, and use that option to render the recipient variables?